wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,139 @@
+# Difficulty Naming
+
+This wiki article serves as a substitute and an update to [this topic](https://osu.ppy.sh/community/forums/topics/178700). 
+
+Difficulty naming is a setting in the song setup that creators are allowed to change to best represent their beatmap's difficulty. As there are many difficulty names, this article contains the ways in which difficulties in each respective game mode are commonly named, as well as various other alternatives that are also allowed.
+
+## Default	
+
+This subsection shows the default scaling difficulty names for each game mode, which are the most commonly used.
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+ 
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy 
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hard](/wiki/shared/diff/hard-s.png "Hard") Hard
+- ![Insane](/wiki/shared/diff/insane-s.png "Insane") Insane
+- ![Expert](/wiki/shared/diff/expert-s.png "Expert") Expert
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Kantan](/wiki/shared/diff/easy-t.png "Kantan") Kantan
+- ![Futsuu](/wiki/shared/diff/normal-t.png "Futsuu") Futsuu
+- ![Muzukashii](/wiki/shared/diff/hard-t.png "Muzukashii") Muzukashii
+- ![Oni](/wiki/shared/diff/insane-t.png "Oni") Oni
+- ![Inner Oni / Ura Oni](/wiki/shared/diff/expert-t.png "Inner Oni / Ura Oni") Inner Oni / Ura Oni
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Cup](/wiki/shared/diff/easy-c.png "Cup") Cup 
+- ![Salad](/wiki/shared/diff/normal-c.png "Salad") Salad
+- ![Platter](/wiki/shared/diff/hard-c.png "Platter") Platter
+- ![Rain](/wiki/shared/diff/insane-c.png "Rain") Rain
+- ![Overdose](/wiki/shared/diff/expert-c.png "Overdose") Overdose
+
+### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![EZ](/wiki/shared/diff/easy-m.png "EZ") EZ 
+- ![NM](/wiki/shared/diff/normal-m.png "NM") NM
+- ![HD](/wiki/shared/diff/hard-m.png "HD") HD
+- ![MX](/wiki/shared/diff/insane-m.png "NX") MX
+- ![SC](/wiki/shared/diff/expert-m.png "SC") SC
+
+## Alternative
+
+The following variants are not commonly used, but they can also be used as a valid alternative way of spread difficulty naming.
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") **Beginner**: in an ENHIX spread, it means the lowest level.
+- ![Advanced](/wiki/shared/diff/normal-s.png "Advanced") **Advanced**: in an ENHIX spread, it's a difficulty that is between Normal and Hard.
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") **Hyper**: in an ENHIX spread, it means a more difficult Hard (also known as Light Insane).
+- ![Extreme](/wiki/shared/diff/expert-s.png "Extreme") **Extreme**: uncommon, but it can be used to represent a difficulty tier above Extra/Expert.
+- ![Marathon](/wiki/shared/diff/expert-s.png "Marathon") **Marathon**: map that is over 5 minutes of [drain time](/wiki/drain_time).
```

Comment:
the way this is structured gives the impression that this is a whole scheme while it isnt

like i think either have whole schemes or dont, the format of the previous default section indicates that the whole article is about naming schemes / conventions so introducing losely described names in an assorted manner is kinda breaking logically with what this thing used to be all about up until this point

Reply:
maybe just move the marathon to global, and the beginner, also don't expect people that read this to be informed enough to know what ENHIX is 0:

Reply:
??? but why? default names are ENHIX not beginner nor marathon